sopel/cli/utils.py

```diff
@@ -327,6 +326,7 @@ def load_settings(options):
     return config.Config(filename)
 
 
+@tools.deprecated(version='7.0', removed_in='8.0')
```

Comment:
Could put a `reason` here, even as simple as "Obsoleted by modernized logging." or something like that. Possibly unnecessary, though, since the Sphinx docs will actually show the details. Your call!